### ISSUE TITLE
Runtime Error

### ISSUE THREAD

**Issue by WaiseFass** [planetxamarin/planetxamarin#645]
**Describe the bug**
The homepage is currently not available.
I tried it in different networks, with different devices

**To Reproduce**
1. Go to https://www.planetxamarin.com/
2. See error

**Expected behavior**
It should be possible to reach the website

**Screenshots**
![image](https://user-images.githubusercontent.com/40561661/75224994-f2a1c900-57a9-11ea-82bd-1a1898421051.png)

**Comment by jfversluis** [comment]
Thanks for letting us know!

I’ll see if this also happens locally so we can maybe fix it by merging new code and trigger a deployment.  Otherwise we need to wait for @jamesmontemagno to dig into logs or a restart

**Comment by jfversluis** [comment]
With the merge of #664 a redeploy happened and everything seems fine now!

**Comment by WaiseFass** [comment]
@jfversluis  server is again not reachable...

**Comment by jfversluis** [comment]
Looks like it's also back up again already! :D

**Comment by Cheesebaron** [comment]
The magic of making an empty commit ;)

**Comment by Cheesebaron** [comment]
@jamesmontemagno can you fetch some logs or something for us to investigate the issue with the site going down all the time? Or perhaps give me access to the resource in your Azure Portal so I can investigate on my own?

**Comment by jamesmontemagno** [comment]
Fixed? idk. it is in internal azure so prob can't give access :(

**Comment by Cheesebaron** [comment]
@jamesmontemagno seems to run right now :)

**Comment by Cheesebaron** [comment]
@jamesmontemagno this still keeps happening. Still relevant to get some logs from the Azure resource so we can investigate where stuff goes wrong.

**Comment by Cheesebaron** [comment]
I disabled those custom errors and now the error is that it cannot load System.IO.Compression.

I've tried bumping to net471 just to see if that helps, also tried explicitly adding that reference through a NuGet package.

Looking in the artifacts folder generated in Azure DevOps, the file is there, and it is matching the signature and version redirected in the Web.config.

I have no idea what is going on. Works fine locally 😢
